packages/train2go-bridge/test/parser.test.js

```
@@ -158,12 +158,14 @@ describe("parser", () => {
 
       // Assert
       expect(activities).toHaveLength(1);
-      expect(activities[0].description).toContain("Avituallamiento intraentreno");
+      expect(activities[0].description).toContain(
```

ESLint

[error] 'expect' is not defined.

(no-undef)